svg/painting/inheritance.svg

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+     xmlns:h="http://www.w3.org/1999/xhtml"
+     width="800px" height="8000px">
```

Comment:
8000px?

Does this assume the UA paints into a buffer the non-visible part?

Reply:
If there are lots of test results, we can scroll down to read them all. There was a similar test I wrote (not uploaded) and I couldn't read all the results when I opened the test .svg in Safari unless I increased the height.